src/__tests__/fixtures/regression.css

```diff
@@ -0,0 +1,7 @@
+@custom-selector :--button .button;
+
+:--button {
+  &--active {
```

Comment:
From this example with the current implementation

```
@custom-selector :--boa-button .boa-button;

&--active {
    color: red;
  }
```

Console output
```
postcss-custom-selectors: /Users/ubi/Sites/open source/boa/src/button/button.css:70:3: The selector ':--boa-button--active' is undefined
```

Output
```
:--boa-button--active {
  color: red
}
```

As you notice is not the correct output at all